cypress/integration/Tabs.spec.ts

```diff
@@ -33,8 +33,8 @@ describe('Tabs', () => {
         cy.findByRole('tab', {name: 'Second Tab'}).should('have.attr', 'aria-selected', 'false');
       });
 
-      it('should not have tabindex on the first tab', () => {
-        cy.findByRole('tab', {name: 'First Tab'}).should('not.have.attr', 'tabindex');
+      it('should not have tabindex=-1 on the first tab', () => {
+        cy.findByRole('tab', {name: 'First Tab'}).should('not.have.attr', 'tabindex', '-1');
```

Comment:
**Before:**

Roving index was generic, but still assumed each item was a `button`. All `button` elements are focusable by default and no `tabIndex` was required. The specification assumed this and made sure not `tabindex` was defined to override the default focusability of the element.

**After:**

The roving focus behavior makes no assumption of the element being used. `tabindex` is always set regardless of inherit focusability status to ensure roving focus always works. So the button now looks like `button[tabindex=0]`. I could have asserted for `tabindex=0`, but it seemed the more generic way of asserting according to the spec was to not override with `tabindex=-1`. Thoughts?